test/cli/files.js

```diff
@@ -127,6 +127,23 @@ describe('files', () => runOnAndOff((thing) => {
       })
   })
 
+  it('add with piped argument', () => {
+    // echo 'src/init-files/init-docs/readme' | jsipfs files add
+    return ipfs('files add', { piped: 'src/init-files/init-docs/readme' })
```

Comment:
Yeah, just got a chance to try it again and this should work:

```
head -c 10 /dev/urandom | base64 | node src/cli/bin.js files add
```

but instead is giving me an error:

```
/Users/user/projects/ipfs/js-monorepo/packages/js-ipfs/node_modules/yargs/yargs.js:1079
      else throw err
           ^

Error: ENOENT: no such file or directory, stat 'Ksw1ReYL8xR2VA=='
    at Error (native)
    at Object.fs.statSync (fs.js:987:18)
    at checkPath (/Users/user/projects/ipfs/js-monorepo/packages/js-ipfs/src/cli/commands/files/add.js:32:10)
    at Object.handler (/Users/user/projects/ipfs/js-monorepo/packages/js-ipfs/src/cli/commands/files/add.js:72:20)
```

Because it's trying to find a file with that name, rather than actually using the data from stdin.